chatgpt.js

```diff
@@ -835,10 +835,9 @@ const chatgpt = {
 
         add: function(instruction, target) {
             const validTargets = ['user', 'chatgpt']; // valid targets
-
             if (!target) return console.error('Please provide a valid target!');
+            if (typeof target !== 'string') return console.error('Target must be a string');
             target = target.toLowerCase(); // lowercase target
-
             if (!validTargets.includes(target))
                 return console.error(`Invalid target ${target}. Valid targets are ${validTargets}`);
 
```

Comment:
It needs to be comprehensive like the routine I gave you for `sendRequest()` i.e. if `instruction` is blank tell user to provide instruction, also there should be `[]` around `${validTargets}`